lab#1/task#4/update-pipeline-definition.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Q: How to work with long params and get it's value
+file=$1
+configmode=$3
+owner=$5
+branch=$7
+poll=$9
```

Comment:
have a look at this: https://www.baeldung.com/linux/bash-parse-command-line-arguments#parsing-long-command-line-options-with-getopt